geoportailv3/static/js/search/searchdirective.js

```diff
@@ -197,17 +198,29 @@ app.SearchDirectiveController =
   var POIBloodhoundEngine = this.createAndInitPOIBloodhound_(
       ngeoCreateGeoJSONBloodhound, searchServiceUrl);
 
-  /** @type {Bloodhound} */
-  var LayerBloodhoundEngine = this.createAndInitLayerBloodhound_();
-
-  /** @type {Bloodhound} */
-  var BackgroundLayerBloodhoundEngine = this.createAndInitLayerBloodhound_();
+  /** @type {Fuse} */
+  var LayerFuseEngine =
+      new Fuse(this.layers_, {
+        keys: ['translatedName'],
```

Comment:
You use a string here to refer to the `translatedName` property, while you use the dot notation elsewhere to access that property on a suggestion instance. That's where your problem is. I'll look at providing a solution to that problem…